test/javascripts/widgets/hamburger-menu-test.js.es6

```diff
@@ -125,42 +128,40 @@ widgetTest("general links", {
   }
 });
 
-widgetTest("category links", {
+widgetTest("top categories - anonymous", {
   template: '{{mount-widget widget="hamburger-menu"}}',
   anonymous: true,
 
-  beforeEach() {
-    const cat = this.site.get("categoriesList")[0];
-
-    const parent = Discourse.Category.create({
-      id: 1,
-      topic_count: 5,
-      name: "parent",
-      url: "https://test.com/parent",
-      show_subcategory_list: true,
```

Comment:
this option is confusing, can you debug and figure out when we currently show subcategories in the hamburger and when we do not?

Reply:
Currently we hiding subcategories [if it's parent category setting `show_subcategory_list` is enabled](https://github.com/discourse/discourse/blob/03a7d532cf8f09b12573b21ef013c21100d52728/app/assets/javascripts/discourse/widgets/hamburger-menu.js.es6#L183-L185). Even though it's not related to the hamburger menu [we using it](https://meta.discourse.org/t/how-to-hide-sub-categories-in-hamburger-menu/58835/5?u=vinothkannans) to reduce the noise in hamburger. Instead we showcasing it in the parent category's landing page (in boxes/rows).